src/entity/model/BellbellPlant_gfx.c

```diff
@@ -0,0 +1,348 @@
+#include "common.h"
+
+extern unsigned char D_0A001090_E6C240[];
+
+Vtx D_0A000000_E6B1B0[] = {
```

Comment:
This one I would definitely make a vtx splat type and #include the inc.c splat creates. also for the non-rectangle vertices below